Rubberduck.RibbonDispatcher/Abstract/RibbonCommon.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using stdole;
+using Microsoft.Office.Core;
+
+using Rubberduck.RibbonDispatcher;
+using Rubberduck.RibbonDispatcher.Abstract;
+
+namespace Rubberduck.RibbonDispatcher.Abstract
+{
+    using ControlSize         = RibbonControlSize;
+    using LanguageStrings     = IRibbonTextLanguageControl;
+    using ChangedEventHandler = EventHandler<ChangedControlEventArgs>;
+
+    public abstract class RibbonCommon : IRibbonCommon {
+        protected RibbonCommon(string id, LanguageStrings strings, bool visible, bool enabled, ControlSize size) {
+            Id               = id;
+            _languageStrings = strings;
+            _visible         = visible;
+            _enabled         = enabled;
+            _size            = size;
+        }
+
+        public event ChangedEventHandler Changed;
+
+        public string       Id          { get; }
+
+        public string       Description => _languageStrings?.Description??"";
+        public string       KeyTip      => _languageStrings?.KeyTip??"";
+        public string       Label       => Use2ndLabel ? _languageStrings?.AlternateLabel??Id 
```

Comment:
"Use2ndLabel" should probably be renamed as "UseAlternateLabel"

Reply:
Also do note that i18n is kind of a solved problem. Why reinvent the wheel here?

Reply:
This is an artifact of the original VBA implementation, where language-specific worksheets were the only available means of storing the i18n text strings. I was waiting until I knew how Rubberduck did i18n to rewrite and redesign this.